src/components/DonutChart/mapFootprintPerTransportTypeToDonutChartData.test.js

```diff
@@ -0,0 +1,42 @@
+import { mapFootprintPerTransportTypeToDonutChartData } from './mapFootprintPerTransportTypeToDonutChartData'
+
+describe('Mapping function api data structure to donut chart data structure', () => {
+  it('should map the transportation type to label and footprintSum to y', () => {
+    expect(
+      mapFootprintPerTransportTypeToDonutChartData({
+        transportationType: 'car',
+        footprintSum: 200,
+      })
+    ).toEqual({ label: 'car', y: 200 })
+  })
+  it('should map empty strings if transportation type is empty', () => {
+    expect(
+      mapFootprintPerTransportTypeToDonutChartData({
+        transportationType: '',
+        footprintSum: 22,
+      })
+    ).toEqual({ label: '', y: '' })
+  })
+  it('should map empty strings if transportation sum is empty', () => {
+    expect(
+      mapFootprintPerTransportTypeToDonutChartData({
+        transportationType: 'car',
+        footprintSum: '',
+      })
+    ).toEqual({ label: '', y: '' })
+  })
+  it('should map empty stringsp if no sum is send', () => {
```

Comment:
ein kleiner Rechtschreibfehler in dieser Zeile und eine Frage - hier könnten doch zwei Gründe das expect bestätigen, einmal wonach du testest (keine sum wird gesendet), aber auch da der transportationType auch ein empty string ist (also wie in Test Zeile 12) müsste ja zu emtpy strings führen? Vielleicht machst du hier keinen empty string in Zeile 31 sondern Daten die eigentlich durchgehen müssten, um wirklich sicher zu sein.

Reply:
Ah du hast Recht! Danke :)